src/Raven.Server/Documents/PeriodicBackup/Retention/GoogleCloudRetentionPolicyRunner.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.Backups;
+using Raven.Server.Documents.PeriodicBackup.GoogleCloud;
+
+namespace Raven.Server.Documents.PeriodicBackup.Retention
+{
+    public class GoogleCloudRetentionPolicyRunner : RetentionPolicyRunnerBase
+    {
+        private readonly RavenGoogleCloudClient _client;
+
+        protected override string Name => "Google Cloud";
+
+        public GoogleCloudRetentionPolicyRunner(RetentionPolicy retentionPolicy, string databaseName, Action<string> onProgress, RavenGoogleCloudClient client)
+            : base(retentionPolicy, databaseName, onProgress)
+        {
+            _client = client;
+        }
+
+        protected override Task<List<string>> GetFolders()
+        {
+            throw new NotSupportedException();
```

Comment:
Create a separate issue to support that.

Reply:
Done